cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -41,6 +41,12 @@ metadata:
   labels:
     tier: node
     component: kube-proxy
+    # Even though kube-proxy runs as a static pod, this annotation lowers the
```

Comment:
I would remove the "Even though kube-proxy runs as a static pod" part, and then add a new sentence at the end "Note that kube-proxy runs as a static pod so this annotation does NOT have any effect on rescheduler (default scheduler and rescheduler are not involved in scheduling kube-proxy)."